src/applications/appeals/10182/containers/IntroductionPage.jsx

```diff
@@ -50,51 +50,53 @@ class IntroductionPage extends React.Component {
+          before February 19, 2019
+        </h5>
+        <p className="vads-u-margin-bottom--0">
+          You'll need to opt in to the new decision review process by checking a
+          box when you fill out the application. This moves your issue from the
+          old appeals process to the new decision review process. As part of the
+          Appeals Modernization Act, our new process means you'll likely get a
+          faster decision.
+        </p>
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
               <h3 className="vads-u-font-size--h4">
-                Determine your eligibility
+                Check to be sure you can request a Board Appeal
               </h3>
-              <p>A board appeal can be requested on:</p>
+              <p>
+                {' '}
+                You can request a Board Appeal up to 1 year from the date on
+                your decision notice. You'll have 60 days if you have a{' '}
+                <a href={CONTESTED_CLAIMS_URL}>contested claim</a> (these are
+                rare).
+              </p>
               <ul>
                 <li>An initial claim decision</li>
                 <li>A supplemental claim decision</li>
                 <li>A Higher-Level Review claim decision</li>
               </ul>
               <p>
-                A board appeal <strong>cannot</strong> be requested on a
-                previous Board Appeal on the <strong>same</strong> claim.
-              </p>
-              <p>
-                You have one year from the date on your decision letter to
-                request a Board Appeal, unless you have a{' '}
-                <a href={CONTESTED_CLAIMS_URL}>contested claim</a> (these are
-                rare).
-              </p>
-              <h4>You’ll have to opt-in to the new decision review process</h4>
-              <p className="vads-u-margin-bottom--0">
-                If you received a VA decision before Febrauary 19, 2019, and you
-                want to appeal your decision under the new decision review
-                process, you’ll need to opt-in. When you fill out the request
-                form, you’ll need to check an opt-in box. This withdraws your
-                claim from the legacy appeal process (the process for decisions
-                received before Febrauary 19, 2019). Your claim will move
-                forward under the new process.
+                <strong>Note: </strong>
+                You can't request a Board Appeal if you've already requested one
+                for this same claim.
               </p>
             </li>
             <li className="process-step list-two">
               <h3 className="vads-u-font-size--h4">Prepare</h3>
-              <p>To fill out this application, you’ll need your:</p>
+              <p>Here's what you'll need to apply:</p>
```

Comment:
Our content likes us to use typographical apostrophe's `’` (or `\u2019`) instead of single quotes.